### PR TITLE
Feature/TFMFE-2828 - Add Support to create presigned post

### PR DESCRIPTION
**Pull request by skoch-tf**

### CODE REVIEW

File: s3upload.js [hunk 1]
```diff
@@ -67,126 +73,172 @@ S3Upload.prototype.createCORSRequest = function(method, url, opts) {
...
-        Object.keys(uploadRequestHeaders).forEach(function(key) {
-            var val = uploadRequestHeaders[key];
-            xhr.setRequestHeader(key, val);
-        });
-    } else {
-        xhr.setRequestHeader('x-amz-acl', 'public-read');
-    }
     this.httprequest = xhr;
-    return xhr.send(file);
+    return xhr.send(formData);
```
Comment (tgomesfork) [inline s3upload.js]: we always send `formData` even if `usePostForm` was not passed?

Comment (skoch-tf) [reply s3upload.js]: yes... here hold the file

File: s3upload.js [hunk 2]
```diff
@@ -67,126 +73,172 @@ S3Upload.prototype.createCORSRequest = function(method, url, opts) {
...
+            Object.keys(uploadRequestHeaders).forEach(key => {
+                const val = uploadRequestHeaders[key];
+                xhr.setRequestHeader(key, val);
+            });
+        } else {
+            xhr.setRequestHeader('x-amz-acl', this.acl);
+        }
+    }
+
+    formData.append('file', file);
```
Comment (skoch-tf) [inline s3upload.js]: @tgomesfork

Comment (teixeirafrancisca) [inline s3upload.js]: out of curiosity: does the put with signedurl also work after this changes?

Comment (skoch-tf) [reply s3upload.js]: no before inside a specific condition, it's a common code

Comment (skoch-tf) [reply s3upload.js]: it's working for both situation, createPresignedPost and signedUrl

**Pull request merged**